src/sol_imgui/sol_imgui.h

```diff
@@ -58,6 +60,18 @@ inline bool BeginChild(const std::string& name, float sizeX, float sizeY, bool b
 {
     return ImGui::BeginChild(name.c_str(), {sizeX, sizeY}, border, flags);
 }
+
+// new
+inline bool BeginChild(const std::string& name, float sizeX, float sizeY, int child_flags)
+{
+    return ImGui::BeginChild(name.c_str(), {sizeX, sizeY}, static_cast<ImGuiChildFlags>(child_flags));
+}
+
+inline bool BeginChild(const std::string& name, float sizeX, float sizeY, int child_flags, int window_flags)
```

Comment:
ImVec2 does not function with sol? i thought i fixed that, would be nice probably to have them exposed also if they work as possible overload.

if not, dont bother, is fine, just an idea (as i believe i made it avalibale outside in some commit)